lib/index.js

```diff
@@ -12,13 +12,15 @@ function AuthPassportLocal () {
   this.context = {};
   this.pipes = pipes;
 
-  this.init = function (config, context, isDummy) {
+  this.init = function (customConfig, context, isDummy) {
     var strategy;
+    const defaultConfig = {
+      'secret': 'changeme',
```

Comment:
A better way to do it is to make this plugin generates a random salt if there is none defined, automatically.
Because users shouldn't need to enter one manually, except if they want strong security (humans are bad at generating random strings).

Now the problem is: where to store it?
A naive approach would be to generate a plain text file in the plugin folder. This works as long as you deploy Kuzzle properly, for instance using a NFS mounting point to share the same files across all your Kuzzle nodes.
But as I see it, a better way to handle this is to have a dedicated plugins storage in Kuzzle, as we discussed a few days ago.

Again, I think we should discuss this before 1.0, but this question does not need an answer immediately and this PR shouldn't be blocked by it.